README.md

```diff
@@ -2,53 +2,26 @@
-nodes can be used to further reduce costs (up to 80% cheaper).
-- **Scale out**: Large numbers of VMs can be requested easily to cope with a burst in jobs.
-
-Despite the cloud focus, all training and model testing works just as well on local compute, which is important for
-model prototyping, debugging, and in cases where the cloud can't be used. In particular, if you already have GPU
-machines available, you will be able to utilize them with the InnerEye toolbox.
-
-In addition, our toolbox supports:
-
-- Cross-validation using AzureML's built-in support, where the models for
-individual folds are trained in parallel. This is particularly important for the long-running training jobs
-often seen with medical images.
-- Hyperparameter tuning using
-[Hyperdrive](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-tune-hyperparameters).
+InnerEye-DeepLearning (IE-DL) is a toolbox for easily training deep learning models on 3D medical images. Simple to run both locally and in the cloud with AzureML, it allows users to train and run inference on the following:
+
+- Segmentation models.
+- Classification and regression models.
+- Any PyTorch Lightning model, via a [bring-your-own-model setup](https://innereye-deeplearning.readthedocs.io/en/latest/docs/bring_your_own_model.html).
+
+In addition, this toolbox supports:
+
+- Cross-validation using AzureML, where the models for individual folds are trained in parallel. This is particularly important for the long-running training jobs often seen with medical images.
+- Hyperparameter tuning using [Hyperdrive](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-tune-hyperparameters).
 - Building ensemble models.
-- Easy creation of new models via a configuration-based approach, and inheritance from an existing
-architecture.
+- Easy creation of new models via a configuration-based approach, and inheritance from an existing architecture.
 
-Once training in AzureML is done, the models can be deployed from within AzureML.
+## Documentation
 
-## Quick Setup
+For all documentation, including setup guides and APIs, please refer to the [IE-DL Read the Docs site](https://innereye-deeplearning.readthedocs.io/en/latest/#).
 
-This quick setup assumes you are using a machine running Ubuntu with Git, Git LFS, Conda and Python 3.7+ installed. Please refer to the [setup guide](docs/environment.md) for more detailed instructions on getting InnerEye set up with other operating systems and installing the above prerequisites.
+## Quick Setup
 
-### Instructions
+This quick setup assumes you are using a machine running Ubuntu with Git, Git LFS, Conda and Python 3.7+ installed. Please refer to the [setup guide](https://innereye-deeplearning.readthedocs.io/en/latest/docs/environment.html) for more detailed instructions on getting InnerEye set up with other operating systems and installing the above prerequisites.
```

Comment:
Can we use local links instead?

Reply:
As they do in the PIP docs: https://pip.pypa.io/en/stable/getting-started/

Reply:
Not in the README no. We could use local links to point them to the GitHub-rendered `environment.md` but the RTD site looks a lot cleaner and I think we should avoid directing anyone to the source docs if possible. 

All roads lead to the docsite!

Reply:
"Local links", if used appropriately, should be links to RTD if the docs are on RTD, but should also work if you build the docs locally. Maybe the term is not the best.